src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -177,21 +182,21 @@ const EncounterList: React.FC<EncounterListProps> = ({
                 onClick={e => {
                   e.preventDefault();
                   if (actionItem.mode == 'edit') {
-                    launchFormInEditMode(
+                    launchEncounterForm(
```

Comment:
This doesn't seem to be passing/accepting a mode compared to the `launchFormInEditMode` won't this bring issues?

Reply:
Yes, this missed me. Updated